src/freenet/node/updater/NodeUpdateManager.java

```diff
@@ -481,12 +485,12 @@ public FreenetURI getSeednodesURI() {
 				"seednodes-" + Version.buildNumber());
 	}
 
-	public FreenetURI getInstallerWindowsURI() {
+	public FreenetURI getInstallerNonWindowsURI() {
```

Comment:
Not a bug fix, there was no bug other than weird inconsistent naming of
methods and constants ;)
Still seemed reasonable to address that. Should have mentioned it in the
commit, though.
Op 25 mrt. 2016 02:36 schreef "Steve Dougherty" <notifications@github.com>:

> In src/freenet/node/updater/NodeUpdateManager.java
> <https://github.com/freenet/fred/pull/516#discussion_r57411463>:
>
> > @@ -481,12 +485,12 @@ public FreenetURI getSeednodesURI() {
> >  				"seednodes-" + Version.buildNumber());
> >  	}
> >
> > -	public FreenetURI getInstallerWindowsURI() {
> > +	public FreenetURI getInstallerNonWindowsURI() {
>
> Heh. Hello bugfix.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/freenet/fred/pull/516/files/f7f60d8805c31051c1df24e9f4968f1ebbe46d03#r57411463>
>